src/components/common/ShortBech32.vue

```diff
@@ -9,6 +10,15 @@
     >
       {{ shortBech32 }}
```

Comment:
suggestion: switch longForm only here instead of copying the whole element. this would reduce code

Reply:
i don't know what you mean.

Reply:
You are copying the whole element
```
    <div
      v-else
      id="address"
      v-clipboard:copy="address"
      v-clipboard:success="() => onCopy()"
      class="address"
    >
      {{ bech32 }}
    </div>
```

This is producing code repetition which should be avoided. Better just switch on the things that actually change, which is only the actual value shown.

Reply:
ah ok! nice suggestion. 

CC @colw

Reply:
@faboweb Ah wait, the reason I copy the whole element is to remove the tool tip on the long form address (it's the same content). It might be a necessary evil.

@jbibla If the tool tip and the copy to clipboard are considered 'tied' together, I could just leave the tool tip in?

Reply:
ah i see. maybe if we leave a tooltip for the long one it should say "Click to Copy". what do you think about that?

@colw i trust you to make the right call - i think it's good either way.